helix-rest/src/main/java/org/apache/helix/rest/client/CustomRestClientFactory.java

```diff
@@ -32,10 +36,13 @@
  */
 public class CustomRestClientFactory {
   private static final Logger LOG = LoggerFactory.getLogger(CustomRestClientFactory.class);
+  private static final int DEFAULT_HTTP_REQUEST_TIMEOUT = 60 * 1000;
```

Comment:
For timeout values, we usually use `long` type.

`60 * 1000L;`

Reply:
I noticed that as well. But since the setSocketTimeout and setConnectionRequestTimeout methods are getting int as input, I prefer to keep them as int. If not we need to have additional type casting.

Reply:
Interesting. In that case, could we leave a comment? That way, the intention is clear, and as the writer of this PR, you can justify using an int.

Reply:
Sure. Added a comments.